utils/create_dummy_models.py

```diff
@@ -378,9 +385,18 @@ def get_tiny_config(config_class, **model_tester_kwargs):
         module_name = model_type_to_module_name(model_type)
         if not modeling_name.startswith(module_name):
             raise ValueError(f"{modeling_name} doesn't start with {module_name}!")
-        module = importlib.import_module(f".models.{module_name}.test_modeling_{modeling_name}", package="tests")
-        camel_case_model_name = config_class.__name__.split("Config")[0]
-        model_tester_class = getattr(module, f"{camel_case_model_name}ModelTester", None)
```

Comment:
Previously, it was doing like:  `Bert[Config] --> Bert[ModelTester]`.
Now, we start to have some model test files that have 2 model test/tester classes. For example: 
- `SpeechT5ModelTest` and `SpeechT5ForSpeechToTextTest`
- `GPTSanJapaneseTest(er)` and `GPTSanJapaneseForConditionalGenerationTest(er)`

In the 2nd case, the logic of [Config] --> [ModelTester] won't find the model tester class for `GPTSanJapaneseConfig` as there is no `GPTSanJapaneseModelTester`.

In general, the current approach is kind limited. This PR tries to make it more general.